src/test/java/csci432/ItemTrackerTest.java

```diff
@@ -0,0 +1,28 @@
+package csci432;
+
+import org.junit.Test;
+import java.awt.image.BufferedImage;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+public class ItemTrackerTest {
+
+    @Test
+    public void insertItemTest() {
+	ItemTracker t = new ItemTracker();
+	//TYPE_INT_ARGB is a placeholder 
+	t.insertItem(new BufferedImage(500,500, BufferedImage.TYPE_INT_ARGB));
```

Comment:
Probably want a way to verify that this method actually works. That may involve checking the size of the queue, or checking that the last item is the item that you just added. Either way we will want a unit test around that